Test: Add getAccuracyClassification function test

BennettElisa opened the pull request:
Wrote various test cases for the redux function **getAccuracyClassification** and changed accuracyCheckPredictedLabels to different values for each test case to ensure the function handles all accurate, mostly accurate, not very accurate and 0% accurate possibilities. 

I also noticed that my eslinter or prettier enjoyed changing all the double quotes to single quotes 🤦🏽 (emoji face palming!) so I'll plan to go back in tomorrow morning and fix it and then submit another pull request but perhaps you can still check over my test specs. Thanks Erin !

Erin007 commented on test/unit/redux.test.js:
```diff
@@ -1,98 +1,231 @@
+    expect(accuracy.percentCorrect).toBe('83.33');
+  });
+
+  test('getAccuracyClassification - not very accuracy', async () => {
```
I think you have a typo -> "not very accura**te**"

Erin007 commented:
YES! This is definitely on the right track, one suggestion on your test set up and yes, please clean-up the styling adjustments from your editor in the redux file.  You'll want to pull and merge staging to resolve the merge conflict as well, please. Let me know if you need a refresher there. :)

BennettElisa commented:
Morning Erin! Styling adjustments have been cleaned and spelling error in test file fixed. Let me know if you see anything else that needs to be addressed!

Erin007 commented:
> Morning Erin! Styling adjustments have been cleaned and spelling error in test file fixed. Let me know if you see anything else that needs to be addressed!

Hmm, @BennettElisa I'm still seeing the styling changes in your PR - any chance you didn't push them? Or your editor settings overwrote them when you committed?  

Can you also address the merge conflict, please. Let me know if you stuck - I'm happy to help.

Erin007 commented on test/unit/redux.test.js:
```diff
@@ -1,98 +1,231 @@
+describe('redux functions', () => {
+  test('getAccuracyClassification - all accurate', async () => {
+    const mockState = resultsStateClassification;
+    mockState.accuracyCheckPredictedLabels = [
```
Can you set your test inputs to named variables, please? Something like `const accurateResults = ['no', 'yes', 'yes', 'yes', 'yes', 'no']` then `resultsStateClassification.accuracyCheckPredictedLabels = accurateResults;`. You'd follow that pattern for each test case. I think it will increase readability, allow you to remove `mockState` and make it more clear what results to expect in each case.

BennettElisa replied:
Yes that makes sense and I agree it will increase readability! I'll double check on the styling changes....perhaps I forgot to push them...hopefully that's the case :) I won't be able to work on it today but I will plan to push changes before  the start of next week.

Erin007 replied:
Awesome! This change looks great. Thank you!

BennettElisa commented:
I believe I removed the merge conflict and also updated the variable names within the test specs for getAccuracyClassification to increase readability . If there is still a merge conflict just let me know and I'll review it again :)

Erin007 commented:
@BennettElisa The tests and merge conflict resolution look great! Thanks for the clean up.  

I think your editor is still doing some undesirable formatting.  For example, I don't expect to see any change to the `redux.js` file, but in your PR there are so many diffs (semicolons, single instead of double quotes etc) that GitHub won't even show them all. The Code.org [setup guide](https://github.com/code-dot-org/code-dot-org/blob/staging/SETUP.md#editor-configuration) has suggestions for editor configuration to auto-format code to adhere to our [style guide](https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md). @made-line might have specific suggestions for VS Code since she worked through something similar recently.

made-line commented:
@BennettElisa Hello! Here are some Prettier settings I've had to change manually in VS Code: 

"semi": "add a semicolon at the end of every line"
"trailingComma": "none" 
"arrowParens": "avoid"

Hope this helps!

BennettElisa commented:
@Erin007 @made-line Thank you both for the suggestions and guidance! I'm working on it now.

BennettElisa commented:
"Closing in favor of #163"
